src/catalog.ts

```diff
@@ -86,7 +90,8 @@ export class Dataset {
     readonly iri: IRI,
     readonly name: string,
     readonly creators: [Organization],
-    readonly distributions: [Distribution]
+    readonly distributions: [Distribution],
+    readonly abbreviation?: string
```

Comment:
In our data model, `abbreviation` makes more sense to me than Schema.org’s more generic `alternateName`.

Reply:
Yes, the alternate name in most instances will be an abbreviation - but not always. For example: the alternate name of `RKDartists` is `RKDartists` (i.e. there is no abbreviation). And the alternate name of `Brinkman Thesaurus` is `Brinkman` (i.e. a shorter version, but not an abbreviation). Would it then make sense to use `alternateName` after all?

Reply:
Good point! It seems an abbreviation can be just [any shorter version](https://en.wikipedia.org/wiki/Abbreviation), so `Brinkman` can really be considered an abbreviation of `Brinkman Thesaurus`. In other words, an abbreviation can be but [doesn’t to be be an acronym](https://www.merriam-webster.com/dictionary/abbreviation) (~ Dutch _afkorting_).

Moreover, it would be nice to communicate to users of this package (among which network-of-terms-comunica) that the secondary name is expected to be shorter than the primary one, so can be used for displaying where space is limited (e.g. in our dropdown 😁).

Is `shortName` an option? I’m also okay with `alternateName` to remain closer to Schema.org; we would then have to document the fact that `alternateName` is a shorter version.

In cases like`RKDArtists`, where short name equals name, would it be clearer to just remove the `alternateName` from the catalog?

Reply:
> In other words, an abbreviation can be but doesn’t to be be an acronym (~ Dutch afkorting).

Valid point! 👍🏻

> Is shortName an option? I’m also okay with alternateName to remain closer to Schema.org; we would then have to document the fact that alternateName is a shorter version.

`shortName` is definitely an option, but we may apply too much meaning to it. The source of the information will eventually be the Registry. If the Registry offers a `shortName` property in its data model, we can use it too (syntactically and semantically). But if the Registry offers an `alternateName` property and we interpret it as a `shortName`, we may be provided with alternate names that aren't short names.

> In cases likeRKDArtists, where short name equals name, would it be clearer to just remove the alternateName from the catalog?

Excellent!